students/13th/jeehoonkim/westagram/user/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from .views      import (
+    SignUpView, 
+    SignInView, 
+    FollowView
+)
+
+urlpatterns = [
+    path('signup', SignUpView.as_view()),
+    path('signin', SignInView.as_view()),
+    path('<int:user_id>/follow', FollowView.as_view())
```

Comment:
```suggestion
    path('/signup', SignUpView.as_view()),
    path('/signin', SignInView.as_view()),
    path('/<int:user_id>/follow', FollowView.as_view())
```
위코드 규칙을 하나 더 알려드리자면, 최상위 url에는 슬래시를 붙이지 않고, 그 다음에 오는 app url의 앞에 슬래시를 붙이는 것을 추천드립니다 :)

Reply:
넵 수정했습니다🙃